lib/private/intercept-exit-callbacks.js

```diff
@@ -271,7 +272,7 @@ module.exports = function interceptExitCallbacks (callbacks, _cache, hash, machi
             // of that exit if available.
             value = new Error('Machine `' + machine.identity + '` called its `' + exitName + '` exit with:\n\n' + util.inspect(value, {depth: null}));
             // Make sure the error stack reaches all the way back to the beginning of the run, following asynchronous hops.
-            value.stack = value.message + '\n' + getCombinedErrorStack(value.stack.split('\n').slice(1), machine._configuredEnvironment.debug.stack);
+            value.stack = value.message + '\n' + getCombinedErrorStack(stripStackMessage(value).split('\n'), machine._configuredEnvironment.debug.stack);
```

Comment:
Can we include the `value.name + ': '+` prefix here? (see comment above on line 810 for explanation)